server/repository/src/migrations/views/mod.rs

```diff
@@ -640,6 +641,35 @@ CREATE VIEW vaccination_course AS
         GROUP BY
 	        invoice_line.invoice_id;
 
+    CREATE VIEW purchase_order_stats AS
+        SELECT
+            po.id AS purchase_order_id,
+            COALESCE(SUM(
+                CASE 
+                    WHEN pol.authorised_number_of_units IS NOT NULL 
+                    THEN pol.authorised_number_of_units * pol.price_per_unit_before_discount
+                    ELSE pol.requested_number_of_units * pol.price_per_unit_before_discount
+                END
+            ), 0) AS line_total_before_discount,
+            COALESCE(SUM(
+                CASE 
+                    WHEN pol.authorised_number_of_units IS NOT NULL 
+                    THEN pol.authorised_number_of_units * pol.price_per_unit_after_discount
+                    ELSE pol.requested_number_of_units * pol.price_per_unit_after_discount
+                END
+            ), 0) AS line_total_after_discount,
+            COALESCE(SUM(
+                CASE 
+                    WHEN pol.authorised_number_of_units IS NOT NULL 
+                    THEN pol.authorised_number_of_units * pol.price_per_unit_after_discount
+                    ELSE pol.requested_number_of_units * pol.price_per_unit_after_discount
+                END
+            ), 0) * (1-(COALESCE(po.supplier_discount_percentage, 0)/100)) AS order_total_after_discount
+        FROM
+            purchase_order po JOIN purchase_order_line pol on po.id = pol.purchase_order_id
```

Comment:
I was thinking this also needs a WHERE clause for deleted datetime - and I realise we don't have it for purchase order lines. 

I guess that is fine? Just flagging here as I don't remember discussing.

Reply:
Good point we don't have `deleted_datetime` on `purchase_order_line` but if we did this would be important...